<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System zarządzania biblioteką</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Dodaj nowego użytkownika</string>
    </property>
    <widget class="QLineEdit" name="nowyUzytkownikImie">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nowyUzytkownikNazwisko">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nowyUzytkownikIndeks">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Imię</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nazwisko</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nr indeksu</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="dodajUzytkownika">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Dodaj</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Usuń użytkownika</string>
    </property>
    <widget class="QLineEdit" name="usunUzytkownikaIndeks">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Nr indeksu</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="usunUzytkownika">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuń</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="usunUzytkownikaPotwierdzenie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Czy chcesz usunąć tego
użytkownika oraz anulować
wszystkie jego wypożyczenia?</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Dodaj nową książkę</string>
    </property>
    <widget class="QLineEdit" name="nowaKsiazkaTytul">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nowaKsiazkaAutor">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Tytuł</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Autor</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="dodajKsiazke">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Dodaj</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nowaKsiazkaKod">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Kod</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Usuń Książkę</string>
    </property>
    <widget class="QLineEdit" name="usunKsiazkeKod">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Kod</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="usunKsiazke">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Usuń</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="usunKsiazkePotwierdzenie">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Czy chcesz usunąć tę książkę
oraz anulować jej
wypożyczenie?</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tabelaUzytkownikow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tabelaKsiazek">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>210</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1102</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
